packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig

```diff
@@ -1,4 +1,5 @@
 {% extends '@ShopsysFramework/Admin/Layout/layoutWithPanel.html.twig' %}
+{% import _self as self %}
```

Comment:
This is little bit weird...I would appreciate at least link to some more information about this but I could not find any relevant :slightly_frowning_face: But maybe you could write a little bit more about it in the commit message - "when there is a macro defined in a template, it can not be used via '__self' in the same template anymore, it must be imported" or something like that

Reply:
updated